src/ln/channel.rs

```diff
@@ -2181,6 +2181,9 @@ impl Channel {
 		if self.channel_outbound {
 			return Err(ChannelError::Close("Non-funding remote tried to update channel fee"));
 		}
+		if (self.channel_state & (ChannelState::ChannelFunded as u32 | ChannelState::RemoteShutdownSent as u32)) != (ChannelState::ChannelFunded as u32) {
```

Comment:
Hmmm if understand well here where is the requirement in the RFCs for receiving node to deny processing of update_fee in case of HTLCs still pending even if remote already sent us shutdown ?